lib/datadog/ci/itr/skippable.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require "json"
+
+require_relative "../ext/transport"
+require_relative "../ext/test"
+
+module Datadog
+  module CI
+    module ITR
+      class Skippable
+        class Test
+          attr_reader :name, :suite
+
+          def initialize(name:, suite:)
+            @name = name
+            @suite = suite
+          end
+
+          def ==(other)
+            name == other.name && suite == other.suite
```

Comment:
Are you missing the bundle (which is the module) here?

Reply:
ruby library does not use modules

Reply:
so I just use the same approach as JS library: only take name and suite from the payload

Reply:
Interesting!